bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/obs/SchedulingBlock.scala

```diff
@@ -30,9 +30,12 @@ object SchedulingBlock {
     apply(start, Some(duration))
 
   def apply(start: Long, duration: Option[Long]): SchedulingBlock =
-    new Impl(start, duration.filter(_ > 0))
+    new Impl(start, duration.filter(_ >= 0))
 
   def unsafeFromStrings(startString: String, durationString: String): SchedulingBlock =
     apply(startString.toLong, durationString.toLong)
 
+  def unsafeFromStrings(startString: String): SchedulingBlock =
```

Comment:
There's no guarantee that the `String` contains a value that can be parsed as a `Long`.  I think these are used from PIO (XML) serialization though so it will all work out unless someone edits the XML.